src/test/java/org/myrobotlab/framework/ConfigTest.java

```diff
@@ -38,33 +32,27 @@
 import org.slf4j.Logger;
 
 public class ConfigTest extends AbstractTest {
-  
-  
+
   @BeforeClass
   public static void setUpBeforeClass() {
-      System.out.println("Runs before any test method in the class");
-  }
-
-  @AfterClass
-  public static void tearDownAfterClass() {
-      System.out.println("Runs after all test methods in the class");
+    // clean out services - reset
+    Runtime.releaseAll(true, true);
```

Comment:
in a perfect world, this wouldn't be necessary.  It's be nice at some point if we have some sort of a warning message that runtime had to release something... that means the previous unit test didn't clean up after itself as expected.

Reply:
I'm not sure, in a perfect world you should be able to verify that the state is "clean"
afaik, the cleanest way is to remove/release everything ... so, its a step closer to idempotent

Reply:
Also, AbstractTest or AbstractServiceTest has reporting on at least straggling threads currently